demo/index.html

```diff
@@ -123,10 +123,13 @@ <h3>Video Player</h3>
             <div class="media-wrapper">
                 <video id="player1" width="640" height="360" style="max-width:100%;" poster="http://www.mediaelementjs.com/images/big_buck_bunny.jpg" preload="none" controls playsinline webkit-playsinline>
                     <source src="https://commondatastorage.googleapis.com/gtv-videos-bucket/CastVideos/mp4/BigBuckBunny.mp4" type="video/mp4">
-                    <track srclang="en" kind="subtitles" src="mediaelement.vtt">
-                    <track srclang="en" kind="chapters" src="chapters.vtt">
+                    <track srclang="de" kind="subtitles" src="german.vtt" label="German with lines">
+                    <track srclang="en" kind="subtitles" src="english.vtt" default>
+                    <track srclang="en" kind="chapters" src="english_chapters.vtt">
+                    <track srclang="de" kind="chapters" src="german_chapters.vtt">
                 </video>
             </div>
+            <div class="myslides"></div>
```

Comment:
If we removed slides, is this `div` even used anymore?

Reply:
DIV was removed